app/pages/sections/catalogue-solutions/order-item/contextCreator.test.js

```diff
@@ -47,43 +52,46 @@ describe('catalogue-solutions order-item contextCreator', () => {
     });
 
     it('should return the questions', () => {
-      const questions = {
-        plannedDate: {
-          id: 'plannedDeliveryDate',
-          mainAdvice: 'Planned delivery date',
-          additionalAdvice: 'For example 14 01 2020',
+      const questions = [{
+        id: 'plannedDeliveryDate',
+        mainAdvice: 'Planned delivery date',
+        additionalAdvice: 'For example 14 01 2020',
+      },
+      {
+        id: 'quantity',
+        mainAdvice: 'Quantity',
+        rows: 3,
+        expandableSection: {
+          dataTestId: 'view-section-quantity-id',
+          title: 'What quantity should I enter?',
+          innerComponent: "Estimate the quantity you think you'll need either per month or per year.",
         },
-        quantity: {
-          id: 'quantity',
-          mainAdvice: 'Quantity',
-          rows: 3,
-          expandableSection: {
-            dataTestId: 'view-section-quantity-id',
-            title: 'What quantity should I enter?',
-            innerComponent: "Estimate the quantity you think you'll need either per month or per year.",
+      },
+      {
+        id: 'selectEstimationPeriod',
+        mainAdvice: 'Estimation period',
+        options: [
+          {
+            value: 'perMonth',
+            text: 'Per month',
           },
-        },
-        estimationPeriod: {
-          id: 'selectEstimationPeriod',
-          mainAdvice: 'Estimation period',
-          options: [
-            {
-              value: 'perMonth',
-              text: 'Per month',
-            },
-            {
-              value: 'perYear',
-              text: 'Per year',
-              checked: true,
-            },
-          ],
-          expandableSection: {
-            dataTestId: 'view-section-estimation-period-id',
-            title: 'What period should I enter?',
-            innerComponent: 'This should be based on how you estimated the quantity you want to order.',
+          {
+            value: 'perYear',
+            text: 'Per year',
+            checked: true,
           },
+        ],
+        expandableSection: {
+          dataTestId: 'view-section-estimation-period-id',
+          title: 'What period should I enter?',
+          innerComponent: 'This should be based on how you estimated the quantity you want to order.',
         },
-      };
+      },
+      {
+        type: 'input',
+        id: 'price',
+      },
+      ];
 
       const context = getContext({ selectedPrice });
       expect(context.questions).toEqual(questions);
```

Comment:
If the questions are getting copied over from the manifest then think it's fair to check the context.questions equals manifest.questions. That way we can remove the `const questions` object